src/pages/Dashboard/Apps/Add/index.jsx

```diff
@@ -263,35 +215,57 @@ export default class AppAdd extends Component {
             >
               {t('Deploy Test')}
             </Button>
-            <Button
-              onClick={() => {
-                if (isCreateApp) {
-                  history.replace(
-                    `/dashboard/app/${appDetail.app_id}/create-version`
-                  );
-                } else {
-                  appCreateStore.reload({ isCreateApp, appId });
-                }
-              }}
-              className={styles.addBtn}
-            >
-              {t('Add delivery type')}
-            </Button>
+            {!isAddVersion && (
+              <Button
+                onClick={() => {
+                  if (isCreateApp) {
+                    history.replace(
```

Comment:
Are you sure to use `history.replace`, not `history.push`?

Reply:
There is add  jugde，previous code used history.replace.